File: jsweet-client/src/main/java/json_schema/Participant.java
package json_schema;

import def.js.Array;

public class Participant extends def.js.Object {

	public Integer getId() {
		return (Integer) $get("id");
	}

	public String getName() {
		return (String) $get("name");
	}

	public String getVersion() {
		return (String) $get("version");
	}

	@SuppressWarnings("unchecked")
	public Array<String> getGameTypes() {
		return (Array<String>) $get("gameTypes");
	}

	public String getAuthor() {
		return (String) $get("author");
	}

	public String getCountryCode() {
		return (String) $get("countryCode");
	}

	public String getProgrammingLanguage() {
		return (String) $get("programmingLanguage");
	}
}

File: robocode2-gui/src/main/kotlin/net/robocode2/gui/model/event/Participant.kt
package net.robocode2.gui.model.event

import net.robocode2.gui.model.comm.Content
import net.robocode2.gui.model.comm.ContentType

class Participant(
    val id: Int,
    val name: String,
    val version: String,
    val gameTypes: Set<String>?,
    val author: String?,
    val countryCode: String?,
    val programmingLanguage: String?
) : Content(type = ContentType.PARTICIPANT.type)